src/main/java/codeu/model/data/MentionList.java

```diff
@@ -0,0 +1,99 @@
+package codeu.model.data;
+
+import codeu.model.store.basic.UserStore;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class MentionList {
+    private Message message;
+    private  UserStore userStore;
+
+    public MentionList(Message message, UserStore userStore){
```

Comment:
I'm going to suggest something weird, but bear with me.  Make the constructor private...